### DISCUSSION TITLE
[Contributing] are there any newbie friendly issues?

### DISCUSSION THREAD

**Discussion by sembooooo** [BurntSushi/ripgrep#2213] [General] [answered]
Hi community,
i have just finished quite  a few chapters in rust book. i want to explore more by working on some open source project.
i am not sure if this is foolish of me but i would like to contribute here.
i have seen some 114 open issues. 
Are there are any newbie friendly contributions which i can do ?

**Comment by BurntSushi** [comment, marked as answer]
Apologies, but I just don't have the time to do that kind of issue triage. I'm sure there are some newbie friendly issues, but I can't spare the time to find them unfortunately.

If you come across an issue and have a question I can try to answer, then you can ask it. I'll answer if I can do so quickly.

**Reply by sembooooo** [reply]
Hey thanks for the quick answer. 
I started looking into your code. Slowly. 
I will ask you questions in case if I don't understand something.
